src/features/InvoiceWorks/components/PdfViewer/EditPdf.jsx

```diff
@@ -84,6 +87,16 @@ export default function EditPdf({
   useAppTitle("Edit Invoice");
   const navigate = useNavigate();
 
+  const {
+    control,
+    handleSubmit,
+    formState: { errors },
+  } = useForm({
+    defaultValues: {
+      title: "",
```

Comment:
I think we should revisit this, since this is not only title, as there can be different fields. We also have to work towards getting the EditLineItem components under react-hook-forms. Created ticket to handle that - https://github.com/earmuff-jam/invoicer/issues/89